docs/01-command-line-reference.md

```diff
@@ -267,7 +267,7 @@ Many options have command line equivalents. In those cases, any arguments passed
                               is unspecified, defaults to rollup.config.js)
 -d, --dir <dirname>         Directory for chunks (if absent, prints to stdout)
 -e, --external <ids>        Comma-separate list of module IDs to exclude
--f, --format <format>       Type of output (amd, cjs, es, iife, umd, system)
+-f, --format <format>       Type of output (amd, cjs, es, iife, umd, system) (default is cjs)
```

Comment:
As we are not listing any defaults here for other options, I would prefer not to list this here as well but just in the list of options for the website.